desktop/extensions-sdk/build/set-up/frontend-extension-tutorial.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Set up an advanced frontend extension
+description: Advanced frontend extension tutorial
+keywords: Docker, extensions, sdk, build
+redirect_from:
+  - /desktop/extensions-sdk/tutorials/react-extension/
+  - /desktop/extensions-sdk/build/set-up/react-extension/
+  - /desktop/extensions-sdk/build/set-up/minimal-frontend-using-docker-cli/
+---
+
+To start creating your extension, you first need a directory with files which range from the extension’s source code to the required extension-specific files. This page provides information on how to set up a simple Docker extension that contains only a UI part.
+
+> Note
+>
+> Before you start, make sure you have installed the latest version of [Docker Desktop](../../../release-notes.md).
```

Comment:
If you want users to get the latest version of Desktop, use the following link: https://www.docker.com/products/docker-desktop/

Reply:
I'll change other pages as well, it makes more sense indeed